browser/main/lib/AwsMobileAnalyticsConfig.js

```diff
@@ -34,8 +34,15 @@ function convertPlatformName (platformName) {
   }
 }
 
+function getSendEventCond () {
+  const isDev = process.env.NODE_ENV !== 'production'
+  const isDisable = !ConfigManager.default.get().amaEnabled
+  const isOffline = !window.navigator.onLine
+  return isDev || isDisable || isOffline
```

Comment:
good work
why don't you use this function on [this line](#diff-7091051d0797267a05e32dd7ac21ad56R10)?

Reply:
thanks for your commenting!
I fixed it on b7e43d9